src/gameLogic/execution/ActionExecutor_Gamble.cpp

```diff
@@ -2,12 +2,31 @@
 // Created by jonas on 28.04.20.
 //
 #include "ActionExecutor.hpp"
+#include "util/GameLogicUtils.hpp"
 
 namespace spy::gameplay {
     bool ActionExecutor::executeGamble(State &s, const GambleAction &op) {
         auto character = s.getCharacters().getByUUID(op.getCharacterId());
         character->subActionPoint();
-        // TODO: implement
-        return false;
+        auto targetField = s.getMap().getField(op.getTarget());
+
+        double winningChance = 18/37;
+        if (character->hasProperty(character::PropertyEnum::LUCKY_DEVIL)) {
+            winningChance = 32/37;
```

Comment:
```suggestion
            winningChance = 32.0/37.0;
```
Ansonsten wird eine Integer-Division ausgeführt und das Ergebnis (0) zu double gecastet